rites/roman1969/src/locales/sk.ts

```diff
@@ -926,8 +926,8 @@ export const locale: Locale = {
     teresa_benedicta_of_the_cross_stein_virgin: 'Svätej Terézie Benedikty od Kríža, panny a mučenice',
     teresa_benedicta_of_the_cross_stein_virgin_copatroness_of_europe:
       'Svätej Terézie Benedikty od Kríža, panny a mučenice, patrónky Európy',
-    teresa_of_calcutta_religious: 'Svätej Terézie z Kalkaty, panny a rehoľníčky',
-    teresa_of_jesus_jornet_ibars_virgin: 'Svätej Terézie od Ježiša Jornet Ibars, panny',
+    teresa_of_calcutta_virgin: 'Svätej Terézie z Kalkaty, panny a rehoľníčky',
+    teresa_of_jesus_jornet_ibars_virgin: 'Svätej Terézie od Ježiša Jornet Ibars, ',
```

Comment:
On lc.kbs.sk (semi-official litirgical calendar based on the Slovak ordo) used to have this celebration inscribed (and so did the hard copy of the ordo), however, last time it was seems to be in [2017](https://lc.kbs.sk/#20170905) as _Svätej Terézie z Kalkaty (Matky Terezy), panny a rehoľnice_ [Saint Teresa of Calcutta (Mother Teresa), Virgin and Religious]. That said, let’s follow the official decree in this case too.

```suggestion
    teresa_of_calcutta_virgin: 'Svätej Terézie z Kalkaty, panny',
    teresa_of_jesus_jornet_ibars_virgin: 'Svätej Terézie od Ježiša Jornet Ibars, panny',
```

Reply:
the second one doesn't have anything to do with this fix. do you have a source for St. Teresa of Jesus Jornet Ibars?

Reply:
hmmm, i think the diff here is desceptive, because the original already had virgin for teresa of jesus, so i'll resolve this one.

Reply:
> the second one doesn't have anything to do with this fix. do you have a source for St. Teresa of Jesus Jornet Ibars?
>
> hmmm, i think the diff here is desceptive, because the original already had virgin for teresa of jesus, so i'll resolve this one.

Yeah, you updated the wrong celebration, I just fixed it. I should have described my changes explixitly, sorry.

I have no source for St Teresa of Jesus Jornet Ibars.